### DISCUSSION TITLE
Comments on source code!

### DISCUSSION THREAD

**Discussion by SylvanFranklin** [ThumusLive/utpm#34] [General]
I want to help out this project and as I was working through the source code trying to understand it I noticed that some parts were missing comments. I would be happy to work on adding those or clarifying existing ones! I want to soon move from working on documentation / readme to working on the actual code part of the project, I'm still trying to get familiar with the code and I feel like better docs can help more people do that more quickly!

**Comment by Thumuss** [comment]
No problem! I'll take time to make proper comments!
